### PR TITLE
Add Breadcrumb component

### PR DESCRIPTION
**Pull request by minsooshin**
- Add Breadcrumb component
  - Increase test coverage
  - Update directory structure to more meaningful

### CODE REVIEW

**Comment by minsooshin** [comment]
screen shot 2017-08-24 at 10 15 06 pm

File: config/webpack.config.dev.js [hunk 1]
```diff
@@ -222,7 +222,7 @@ module.exports = {
             loader: 'css-loader',
             options: {
               modules:true,
-              localIdentName: '[local]-[hash:base64:5]'
+              localIdentName: '[local]'
```
Comment (minsooshin) [inline config/webpack.config.dev.js]: changed to original class name since the hash is not meaningful

**Comment by minsooshin** [comment]
screen shot 2017-08-24 at 10 20 14 pm

**Comment by minsooshin** [comment]
screen shot 2017-08-24 at 10 46 47 pm

**Comment by minsooshin** [comment]
screen shot 2017-08-25 at 11 39 08 am

File: src/components/Breadcrumb/index.jsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    {Children.map(children, (child, i) => {
+      return (
+        <ListItem key={i} className={classNames(className.list)}>
+          {child}
+          {i !== children.length - 1 &&
+            <Icon
+              type={iconStyle.type}
+              name={iconStyle.name}
+              size={iconStyle.size}
+              className={classNames('Breadcrumb__icon', className.icon)} />
```
Comment (minsooshin) [inline src/components/Breadcrumb/index.jsx]:
```jsx
<Icon
  className={classNames('Breadcrumb__icon, className.icon)}
  {...iconStyle} />
```

File: src/components/Breadcrumb/index.jsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+const classNames = cx.bind(style)
+
+const Breadcrumb = ({ className, icon, children }) => {
+  const iconStyle = Object.assign({}, {
+    type: 'materialDesign',
+    name: 'MdNavigateNext',
+    size: 20,
+  }, icon)
+  return (<List type="horizontal" className={classNames('Breadcrumb', className.root)}>
```
Comment (minsooshin) [inline src/components/Breadcrumb/index.jsx]: Doesn't need `Breadcrumb` class
```jsx
return (<List type="horizontal" className={classNames(className.root)}>
```

**Comment by RaulZhao** [comment]
+1

**Review by RaulZhao — APPROVED** [review]

**Pull request merged**